### PR TITLE
Update Kafka to 1.0.0

### PR DESCRIPTION
**Pull request by scholzj**

### CODE REVIEW

**Comment by scholzj** [comment]
I checked the updated version against:
* The Debezium guide using Barnabas
* EnMasse / AMQP - Kafka bridge

All seems to work fine. Anything else comes to your mind before we merge this?

File: kafka-base/Dockerfile [hunk 1]
```diff
@@ -10,20 +10,20 @@ RUN groupadd -r -g 1001 kafka && useradd -r -m -u 1001 -g kafka kafka
 
 # Set Scala and Kafka version
 ENV SCALA_VERSION=2.11
-ENV KAFKA_VERSION=0.11.0.1
+ENV KAFKA_VERSION=1.0.0
 
 # Set Kafka MD5checksum
```
Comment (tombentley) [inline kafka-base/Dockerfile]: I think we changed to sha512 before this PR, but we should fix this comment.

Comment (scholzj) [reply kafka-base/Dockerfile]: Yeah, we changed the checksum before. I already changed the filename in this PR, but it seems I didn't noticed the comment. Should be fixed now.

**Comment by scholzj** [comment]
@tombentley I haven't changed anything in the bridge. I just deployed it with EnMasse and used it to send / receive messages. Nothing else. So yes, it used the old protocol.

**Comment by tombentley** [comment]
OK, thanks @scholzj.

lgtm, but let's wait for @ppatierno before merging.

**Comment by scholzj** [comment]
@tombentley Yeah, I promised him that he can *press the button* tomorrow. Its all yours now @ppatierno :-)

**Pull request merged**